client/apps/web/src/components/Sidebar.tsx

```diff
@@ -141,16 +136,17 @@ export const Sidebar = () => {
   const router = useRouter()
   const searchParams = useSearchParams()
   const pathname = usePathname()
-  const params = useParams<{ orgKey: string }>()
-  const pathWithoutParams = pathname.split(params.orgKey)[0]
+  const org = pathname.split("/")[1]
```

Comment:
일관성 있게 
```suggestion
  const { orgKey } = useParams<{ orgKey: string }>()
```
로 하는 건 어떨까요?